src/main/java/org/haffson/adventofcode/utils/ProblemStatus.java

```diff
@@ -0,0 +1,21 @@
+package org.haffson.adventofcode.utils;
+
+import org.haffson.adventofcode.ProblemStatusEnum;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ProblemStatus {
+
+    private ProblemStatus() {
+    }
+
+    public static Map<Integer, ProblemStatusEnum> getProblemStatusMap(int part1, int part2,
+                                                                      ProblemStatusEnum status1, ProblemStatusEnum status2) {
```

Comment:
Ich nutze gerne folgenden Stil, weil er etwas übersichtlicher ist. Ist aber natürlich geschmacksache.
```suggestion
    public static Map<Integer, ProblemStatusEnum> getProblemStatusMap(
        int part1,
        int part2,
        ProblemStatusEnum status1,
        ProblemStatusEnum status2
    ) {
```